dbms/src/Flash/Mpp/MPPTunnelSet.h

```diff
@@ -71,6 +72,8 @@ class MPPTunnelSetBase : private boost::noncopyable
 
 private:
     std::vector<TunnelPtr> tunnels;
+    std::unordered_map<MPPTaskId, size_t> id_to_index_map;
```

Comment:
```suggestion
    std::unordered_map<MPPTaskId, size_t> target_id_to_index_map;
```

Reply:
Good idea, but the pr was already merged :(